msgraph-mail/dotnet/Models/Odata/Error/Error.cs

```diff
@@ -0,0 +1,35 @@
+using Microsoft.Kiota.Abstractions.Serialization;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+namespace Graphdotnetv4.Models.Odata.Error {
+    public class Error : Exception, IParsable {
```

Comment:
It feels like this class should be called something like `ErrorResponse`. Then `Error_prop` type be called `Error` instead of "Main".

Reply:
This comes from the open api description and can be fixed in there